Automattic/wp-calypso
# Add Related Plugins page

Closes #80289

## Proposed Changes

- create a new route: `/plugins/:plugin/related/:siteId`
- display related plugins on the new route, similarly to the category pages
- use the related plugins data from #80287
- reuse the `normalizePluginData` function to map fields from ES result, related diff: D121300-code

## Testing Instructions



* Apply the diff D121300-code to your sandbox
* Visit `/plugins/:plugin/related` with any plugin (eg. `woocommerce`)
* Check that the plugin cards contain all the necessary information, and clicking them redirect to the plugin pages
  <img width="1072" alt="CleanShot 2023-09-12 at 10 27 28@2x" src="https://github.com/Automattic/wp-calypso/assets/11555574/2aab5fcf-7af5-4561-8e34-31d911c4eab1">
* Check that the related plugins component doesn't have any regressions (on the plugin details pages)
* Check that clicking the `See all` link on the related plugins component redirect to the related plugins page
  <img width="608" alt="CleanShot 2023-09-12 at 10 28 27@2x" src="https://github.com/Automattic/wp-calypso/assets/11555574/0239bcde-a7fa-43d9-b1a8-02f90cda5f2c">


## Pre-merge Checklist
